.gitignore

```diff
@@ -3,3 +3,4 @@
 /clover.xml
 /coveralls-upload.json
 /vendor/
+.idea/
```

Comment:
Please revert this because it can be added to your [user global git ignore rules](https://git-scm.com/docs/git-config#Documentation/git-config.txt---global).

Reply:
Done. Thank you for the review @froschdesign